migrations/2024-09-24-205353_uploads/up.sql

```diff
@@ -0,0 +1,8 @@
+CREATE TABLE uploads (
```

Comment:
would help with replayability and iterating/testing:
```suggestion
CREATE TABLE IF NOT EXISTS uploads (
```

Reply:
This isn't necessary since the migration is not meant to be idempotent. The `down` migration is there to support `diesel migration revert` and `diesel migration redo` for replayability / reconstruction of the database schema.
<img width="646" alt="image" src="https://github.com/user-attachments/assets/b532fa25-11aa-473e-b883-a3759015f685">